src/components/molecules/Avatar/component.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC, useCallback, useEffect, useState } from 'react';
+import { Props } from './props';
+import styled from '@emotion/styled';
+
+export const Avatar: FC<Props> = ({ url, alt, ...rest }: Props) => {
+  return(
+    <div {...rest}>
+      <img src={url} alt={alt}/>
```

Comment:
What if Image is not a square dimensioned? Avatar has to always be a circle shaped. Also, make image a block, due to it's default line-height issue.